packages/ui/CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 ### Deprecated
 
+- `ui/IconButton` to be removed in the future
```

Comment:
let's put "to be removed in 5.0.0" for all these components

Reply:
Done